command/git.go

```diff
@@ -55,37 +55,18 @@ func (c *gitFetchProject) ParseParams(params map[string]interface{}) error {
 			"must not be blank", c.Name())
 	}
 
-	if strings.HasPrefix(c.Token, "token") {
-		splitToken := strings.Split(c.Token, " ")
-		if len(splitToken) != 2 {
-			return errors.New("token format is invalid")
-		}
-		c.Token = splitToken[1]
-	}
 	return nil
 }
 
-func buildHTTPCloneCommand(location *url.URL, branch, dir, token string) ([]string, error) {
+func buildHTTPCloneCommand(location *url.URL, owner, repo, branch, dir, token string) ([]string, error) {
```

Comment:
does it make sense, given 5 string arguments, to package this up as a struct, or methods on a type?

Reply:
made it a struct